content/tutorials/wallets.md

```diff
@@ -19,13 +19,74 @@ The private key is used to spend the Ocean Tokens in the account. You must keep
 
 ## Wallet Options
 
-Ocean Tokens are [ERC-20 tokens](https://en.wikipedia.org/wiki/ERC-20), so any wallet that supports arbitrary ERC-20 tokens should work to hold Ocean Tokens.
+Ocean Tokens are [ERC-20 tokens](https://en.wikipedia.org/wiki/ERC-20), so any wallet that supports arbitrary ERC-20 tokens should work to hold Ocean Tokens. Any standard Ethereum-compatible tokens are called ERC-20 tokens.
+
+There are many kinds of wallets (e.g. paper wallets, hardware wallets, software wallets, custodial wallets), each with its own advantages and disadvantages. There is a tradeoff between security and convenience. We encourage you to search around and read about wallets to understand the options. In this section, we will explain how to setup a create a MetaMask wallet account. 
+
+
+## Software Wallet
+### MetaMast Setup
+
+We will be walking you through a tutorial to setup Metamask, if you haven't done so already.
+
+**What is MetaMask?**
+
+MetaMask is a browser extension that allows web applications to interact with the Ethereum blockchain. In our current setup, browsers like Chrome/Firefox display information by fetching it from a database. But our current web browsers (web 2.0 as it’s called) are not build to interface with distributed databases a.k.a. the blockchain (web 3.0). This is why MetaMask is needed, as it allows modern web browsers to interface with the Ethereum blockchain.
+
+
+**Why is Metamask required?**
+
+The unique part about MetaMask is that it serves a dual purpose as an ERC-20 wallet and a Web 3.0 browser. For users, it works as an Ethereum wallet, allowing them to store and send any ERC-20 tokens. For developers, it allows you to design and run Ethereum DApps right in your browser without running a full Ethereum node. MetaMask talks to the Ethereum blockchain for you.
+
+**Setup Instructions for Metamask**
+1. Go to the Chrome Web Store for extensions and search/install metamask.
+
+![metamask-chrome-store](https://lh5.googleusercontent.com/lWV39nXsUmTsoilfQMGjWbe7VRzO2FciSrDGZOsTqTBGWvFH8YanYLtzuRw=w2400)
```

Comment:
please put all those images into the `tutorials/images` folder and include them from there instead of linking to remote Google resources. Only this will generate proper image sizes

Reply:
@Anirudh2490 this solves your image resizing issue